.travis.yml

```diff
@@ -29,3 +32,5 @@ after_success:
   # gover collects all .coverprofile files and saves it to one file gover.coverprofile
   - gover
   - goveralls -coverprofile=gover.coverprofile -service=travis-ci
+  # sync the docs only if everything else was successful
+  - cd script && ./sync-docs.sh
```

Comment:
if you think it will be safer like this, than i'm ok with it.

Maybe just one thing to consider: If another command is added to list after `cd script && ./sync-docs.sh` what is its cwd going to be? `./script/` or it will remain as before?